service/chartconfig/v1/appr/appr.go

```diff
@@ -48,16 +49,17 @@ func New(config Config) (*Client, error) {
 		Timeout: time.Second * httpClientTimeout,
 	}
 
-	u, err := url.Parse(config.Address + "/" + config.Organization + "/")
+	u, err := url.Parse(config.Address + "/cnr/api/v1/")
```

Comment:
Just out of interest: Did paths already change in the interface?

Reply:
Because of the integration test :) Previously we only had a unit test where we setup both the client and the server and all worked fine of course, after trying to access a real server the actual path used needed to be fixed.